project_internal_access_from_portal/demo/demo_data.xml

```diff
@@ -0,0 +1,32 @@
+<odoo noupdate="1">
+  <record id="demo_portal_user_partner" model="res.partner">
+    <field name="name">Demo Portal User</field>
+    <field name="email">demo_portal@example.com</field>
+    <field name="is_company" eval="False" />
+  </record>
+
+  <record id="demo_portal_user" model="res.users">
+    <field name="name">Demo Portal User</field>
+    <field name="partner_id" ref="demo_portal_user_partner" />
+    <field name="login">demo_portal@example.com</field>
+    <field name="password">demo</field>
+    <field name="groups_id" eval="[(4, ref('base.group_portal'))]" />
+  </record>
+
+  <record id="demo_portal_internal_project" model="project.project">
+    <field name="name">Demo Internal/Portal Project</field>
+    <field name="privacy_visibility">portal_internal</field>
+    <field name="message_follower_ids" eval="[(4, ref('demo_portal_user_partner'))]" />
```

Comment:
Please check which records are in the `message_follower_ids` field before using it.